src/docs/Components/DataTable/index.mdx

```diff
@@ -70,6 +96,34 @@ The Datatable allows to display several types of data.
 | **KPI**                               | Display a progression indicator                                                                                               | Adapted to the content |
 | **Expand**                            | Display sub-items or details about an item (Only one column per table, always in first position or after the checkbox column) | Fixed                  |
 
+#### Rows
+
+In a table, a row corresponds to an item.  
+The columns can then be used to list different types of data related to that particular object such as status, description or numeric values.  
+A row cannot be selected without a checkbox. Once selected, the row changes its state, displaying a selected checkbox and a different colour background.
```

Comment:
~~A row cannot be selected without a checkbox. Once selected, the row changes its state, displaying a selected checkbox and a different colour background.~~ A row must be selected with a checkbox, and each selected row turns to a different colour background to make it easier to read.